### PR TITLE
Pass the "--names" option to PHPCPD.

### PR DESCRIPTION
**Pull request by beerendlauwers**
Addresses https://github.com/EdgedesignCZ/phpqa/issues/170.

### CODE REVIEW

**Comment by zdenekdrahos** [comment]
It might be better to use [`phpqa.extensions`](https://github.com/EdgedesignCZ/phpqa/blob/v1.23.3/.phpqa.yml#L17) just like in the [other tools](https://github.com/EdgedesignCZ/phpqa/blob/v1.23.3/src/Tools/Analyzer/Phpcs.php#L28). The extensions are already [documented](https://github.com/EdgedesignCZ/phpqa#advanced-configuration---phpqayml) and you don't have to define extensions twice.  What you think?

```php
$phpcpdNames = array_map(
    function ($extension) {
        return "*.{$extension}";
    },
    explode(',', $this->config->csv('phpqa.extensions'))
);
// ...
names' => implode(',', $phpcpdNames),
```

**Comment by beerendlauwers** [comment]
Good point. Refactored.

File: src/Tools/Analyzer/Phpcpd.php [hunk 1]
```diff
@@ -13,12 +13,20 @@ class Phpcpd extends \Edge\QA\Tools\Tool
...
+            explode(',', $this->config->csv('phpqa.extensions'))
+        );
+
         $args = array(
             'progress' => '',
             $this->options->ignore->bergmann(),
             $this->options->getAnalyzedDirs(' '),
             'min-lines' => $this->config->value('phpcpd.minLines'),
             'min-tokens' => $this->config->value('phpcpd.minTokens'),
+            'names' => $phpcpdNames,
```
Comment (zdenekdrahos) [inline src/Tools/Analyzer/Phpcpd.php]: Please fix [CI](https://travis-ci.org/EdgedesignCZ/phpqa/jobs/620694261#L573).

```suggestion
        );
        $phpcpdNames = array_map(
            function ($extension) {
                return "*.{$extension}";
            },
            array_filter(explode(',', $this->config->csv('phpqa.extensions')))
        );
        if ($phpcpdNames) {
            $args['names'] = \Edge\QA\escapePath(implode(',', $phpcpdNames));
        }
```

**Comment by zdenekdrahos** [comment]
Fixed in https://github.com/EdgedesignCZ/phpqa/pull/193

**Pull request closed without merging**